tests/app/notifications/test_process_notification.py

```diff
@@ -356,7 +356,7 @@ def test_persist_notification_increments_cache_if_key_exists(sample_template, sa
             "sms",
             "normal",
             "+14383898585",
-            "notify-internal-tasks",
+            "send-throttled-sms-tasks",
```

Comment:
Does not care that a custom queue was requested, still routed to the throttled queue